code/modules/surgery/organs/tongue.dm

```diff
@@ -25,7 +25,15 @@
 		/datum/language/vampiric,
 		/datum/language/dwarf,
 		/datum/language/vox, //Skyrat change - adds vox language
-		/datum/language/machine //Skyrat change - adds machine language
+		/datum/language/machine, //Skyrat change - adds machine language
+		/datum/language/calcic, //Skyrat change - plasmaman language
+		/datum/language/dunmeri, //Skyrat change - dunmer language
+		/datum/language/moffic, //Skyrat change - insect language
+		/datum/language/neokanji, //Skyrat change - generally asian language
+		/datum/language/shadowtongue, //Skyrat change - shadowpeople language
+		/datum/language/solcommon, //Skyrat change - sol common blah blah
+		/datum/language/sylvan, //Skyrat change - plantpeople langauge
+		/datum/language/technorussian //Skyrat change - russian sterytype language
```

Comment:
Spell Check (Yes this is the only one i checked, i only care about Russian)

Reply:
fix xed